webpage/index.rst

```diff
@@ -1,42 +1,31 @@
 .. include:: references.rst
 
-gamma-cat
-=========
+******************************************************************************
+Gamma-Cat - An open data collection and source catalog for gamma-ray astronomy
+******************************************************************************
 
-**gamma-cat** is an open data collection and source catalog for gamma-ray astronomy.
+Gamma-Cat is an open data colleciton and source catalog for currently 168 TeV gamma-ray sources.
+This page (http://gamma-cat.readthedocs.io) contains the Gamma-Cat documentation.
+It provides an introduction to Gamma-Cat, see :ref:`introduction`, a description of accessing the catalog and collection themselves, see :ref:`datadocs`, a documentation for users, see :ref:`userdocs`, and for contributors, see :ref:`contributordocs`.
 
-* Data and Docs (this site): |gamma-cat-docs|
-* Repository: |gamma-cat-repo|
-* Explore `gamma-cat` and other gamma-ray data at `gamma-sky.net`_
-* Access and analyse `gamma-cat` and other gamma-ray data from Python: `gammapy.catalog`_
-
-This site is organised in the following sections:
-
-* :ref:`about-docs` - Quick introduction to `gamma-cat`
-* :ref:`user-docs` - Detailed info about `gamma-cat`
-* :ref:`contributor-docs` - How to give feedback and contribute
+.. toctree::
+   :maxdepth: 3
 
-.. _about-docs:
+   intro/index
 
 .. toctree::
-   :maxdepth: 1
-   :caption: About
+   :maxdepth: 3
 
-   about
-
-.. _user-docs:
+   data/index
 
 .. toctree::
    :maxdepth: 1
-   :caption: User Documentation
 
    use/index
-   use/source_list
-   use/reference_list
    stats/index
```

Comment:
What about this "stats" and "changes" parts. They are looking not very well but I don't know what the purpose is or was.